### PR TITLE
Changing static HTML to Jinja2

### PR DESCRIPTION
**Pull request by Pacu2**

### CODE REVIEW

**Comment by Pacu2** [comment]
https://kindergarten.stxnext.pl/browse/WDP-59

File: src/presence_analyzer/static/js/mean_time_weekday.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            var selected_user = $("#user_id").val(),
+                chart_div = $('#chart_div');
+            if(selected_user) {
+                loading.show();
+                chart_div.hide();
+                $.getJSON("/api/v1/mean_time_weekday/"+selected_user, function(result) {
+                    $.each(result, function(index, value) {
+                        value[1] = parseInterval(value[1]);
+                    });
+                    console.log(result);
```
Comment (dpolcyn) [inline src/presence_analyzer/static/js/mean_time_weekday.js]: ? ;)

File: src/presence_analyzer/views.py [hunk 2]
```diff
@@ -84,3 +85,14 @@ def presence_start_end_time(user_id):
         (calendar.day_abbr[weekday], start, end)
         for weekday, (start, end) in enumerate(weekdays)
     ]
+
+
+@app.route('/<string:template_name>')
+def render_view(template_name):
+    """Render template based on html file"""
+    if not template_name.endswith(".html"):
+        template_name += ".html"
```
Comment (dpolcyn) [inline src/presence_analyzer/views.py]: use format

File: src/presence_analyzer/views.py [hunk 3]
```diff
@@ -84,3 +85,14 @@ def presence_start_end_time(user_id):
         (calendar.day_abbr[weekday], start, end)
         for weekday, (start, end) in enumerate(weekdays)
     ]
+
+
+@app.route('/<string:template_name>')
+def render_view(template_name):
+    """Render template based on html file"""
```
Comment (dpolcyn) [inline src/presence_analyzer/views.py]: Full stop on the end of doc

File: src/presence_analyzer/views.py [hunk 4]
```diff
@@ -44,11 +46,10 @@ def mean_time_weekday_view(user_id):
...
 
     weekdays = group_by_weekday(data[user_id])
-    result = [
+    return [
         (calendar.day_abbr[weekday], mean(intervals))
-        for weekday, intervals in enumerate(weekdays)
-    ]
-    return result
+        for weekday, intervals in weekdays.items()
+        ]
```
Comment (dpolcyn) [inline src/presence_analyzer/views.py]: ] to far

File: src/presence_analyzer/views.py [hunk 5]
```diff
@@ -84,3 +85,14 @@ def presence_start_end_time(user_id):
         (calendar.day_abbr[weekday], start, end)
         for weekday, (start, end) in enumerate(weekdays)
     ]
+
+
+@app.route('/<string:template_name>')
+def render_view(template_name):
+    """Render template based on html file."""
+    if not template_name.endswith(".html"):
+        "{}.html".format(template_name)
```
Comment (dpolcyn) [inline src/presence_analyzer/views.py]: '

**Pull request merged**